### DISCUSSION TITLE
keyball 44 USBが反応しない

### DISCUSSION THREAD

**Discussion by kr545** [Yowkees/keyball#616] [Q&A]
はんだ付け初心者なりに試行錯誤してみたのですが、手詰まりになってしまったのでご質問させてください。　

先日 Keyball 44を購入し、以下の手順ではんだ付けを行っていたのですがUSBの赤いランプが点灯せず困っています。可能であれば解決策をご教授していただけると幸いです。

1. ビルドガイドに沿ってはんだ付けを行う
2. 左右のpro microに Keyball_test.hex を書き込む
3. 左右のpro microのＯＬＥＤが点灯しない
4. はんだ付けのやり直し
5. 左側のＯＬＥＤの点灯&キー入力の確認
6. 右側（トラックボール側）のＵＳＢランプが点灯しない&キー入力ができない

はんだ付けを何度もやり直したので、もしかしたらどこか破損させてしまったのかもしれません。
※トラックボール側のＯＬＥＤが少し焼けてしまいましたが、左側の基板に差し込むと点灯するのでＯＬＥＤ自体に問題はなさそうです。

よろしくお願いいたします。

![IMG_3011](https://github.com/Yowkees/keyball/assets/97976292/d5b91376-57cd-4f99-8419-f7d7564d2b34)
![IMG_3012](https://github.com/Yowkees/keyball/assets/97976292/f39dc7a8-068c-4ffc-86bf-ea0d0cefaf2b)
![IMG_3008](https://github.com/Yowkees/keyball/assets/97976292/b0bda0db-8471-4670-965b-3fb1d2cdcada)

**Comment by Ryo624** [comment]
特にTRRSジャックやリセットボタンあたりですがProMicroやキーソケットなど全体的にはんだの量が多すぎる気がします。
はんだの量が多すぎると逆に端子に上手く馴染まずに接触不良になったり、配線の上にはんだがのってしまうとショートの原因になってしまうので、はんだの量を減らして形が山型になるように端子に馴染むことを意識してはんだ付けをやり直してみてください。

**Reply by kr545** [reply]
ありがとうございます。

はんだの量を減らして接続してみたところ、無事にＵＳＢの認識とキー入力ができました。
引き続きkeyballの完成を目指していこうと思います。

**Reply by Ryo624** [reply]
ご対応いただきましてありがとうございます。無事に解決したようで良かったです。
